x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.tsx

```diff
@@ -39,15 +43,20 @@ import { getFieldConfig } from '../../../lib';
 import { useAppContext } from '../../../../../app_context';
 import { Form, UseField, useForm } from '../../../shared_imports';
 import { useLoadInferenceModels } from '../../../../../services/api';
+import { getTrainedModelStats } from '../../../../../../hooks/use_details_page_mappings_model_management';
+import { InferenceToModelIdMap } from '../fields';
+
 interface Props {
   onChange(value: string): void;
   'data-test-subj'?: string;
   setValue: (value: string) => void;
+  setNewInferenceEndpoint: (newInferenceEndpoint: InferenceToModelIdMap) => void;
```

Comment:
We added a new prop here. It would be great if we can some tests to this [file](https://github.com/elastic/kibana/blob/9cb7940601af359c624d70f6ff158f5e156f5861/x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.test.tsx#L37-L40) for this change.